--- dataverse_header.xhtml.orig	2016-12-14 18:15:30.597000000 -0500
+++ dataverse_header.xhtml	2016-12-14 20:57:22.949000000 -0500
@@ -21,7 +21,10 @@
                         <span class="icon-bar"></span>
                     </button>
                     <a href="/">
-                        <span class="navbar-brand"><i id="icon-dataverse" class="icon-dataverse"></i> #{bundle.dataverse}</span>
+                        <span class="navbar-brand"><i id="icon-dataverse" class="icon-dataverse"></i>
+                            <span class="#{bundle.instanceNameFullClasses}"> #{bundle.instanceNameFull}</span>
+                            <span class="#{bundle.instanceNameShortClasses}"> #{bundle.instanceNameShort}</span>
+                        </span>
                     </a>
                     <h:outputLink value="#" onclick="PF('statusInfo').show()" rendered="#{!empty settingsWrapper.get(':StatusMessageText')}">
                         <span class="label label-default status">#{!empty settingsWrapper.get(':StatusMessageHeader') ? settingsWrapper.get(':StatusMessageHeader') : bundle['header.status.header']}</span>
