--- dataverse_template.xhtml	2017-01-28 17:42:24.108718494 -0500
+++ dataverse_template.xhtml	2017-01-26 04:45:38.498692700 -0500
@@ -36,7 +36,17 @@
     <h:body styleClass="#{widgetWrapper.widgetView ? 'widget-view' : ''}">
         <f:loadBundle basename="Bundle" var="bundle"/>
         <a href="#content" class="sr-only">#{bundle['body.skip']}</a>       
+
+	<c:if test="#{bundle.instance_branding_header}">
+            <ui:include src="instance_header.xhtml" rendered="#{bundle.instance_branding_header}"/>
+	</c:if>
+
         <ui:include src="dataverse_header.xhtml">
+
+	<c:if test="#{bundle.instance_branding_footer}">
+            <ui:include src="instance_footer.xhtml"/>
+	</c:if>
+
             <ui:param name="dataverse" value="#{dataverse != null ? dataverse : dataverseServiceBean.findRootDataverse()}"/>
             <ui:param name="showDataverseHeader" value="#{showDataverseHeader != null ? showDataverseHeader : true}"/>
             <ui:param name="showMessagePanel" value="#{showMessagePanel != null ? showMessagePanel : true}"/>
